install/clax_install.sh

```diff
@@ -68,15 +68,16 @@ function generate_inetd {
     echo "clax stream tcp nowait $username $dir/clax clax -l $dir/clax.log" > $dir/inetd.clax
     echo "Generating xinetd.clax file"
     cat >$dir/xinetd.clax <<EOX
-service clax {
```

Comment:
To clarify, xinetd agent throws an error with the `{` in the same line and the service doesn't starts, which is fixed putting the { in the new possition:

`May 12 23:07:12 raspberrypi xinetd[1045]: Reading included configuration file: /etc/xinetd.d/clax [file=/etc/xinetd.d/clax] [line=28]
May 12 23:07:12 raspberrypi xinetd[1045]: Service clax: missing '{' [file=/etc/xinetd.d/clax] [line=2]`